src/compiler/types.ts

```diff
@@ -293,6 +293,8 @@ module ts {
         Const =             0x00001000,  // Variable declaration
         OctalLiteral =      0x00002000,
 
+        SimpleArrowFunction =      0x00004000,  // Arrow function without parenthesized parameters

```

Comment:
@DanielRosenwasser : Simple in this case is has no parenthesis. I don't think you can just check these three conditions. 
@JsonFreeman : That would work too